dataikuapi/dss/managedfolder.py

```diff
@@ -108,6 +109,21 @@ def put_file(self, path, f):
                 "POST", "/projects/%s/managedfolders/%s/contents/%s" % (self.project_key, self.odb_id, utils.quote(path)),
                 "", f)
 
+    def upload_folder(self, path, folder):
+        """
+        Upload folder and its content as path in the managed folder.
+
+        :param str path: the destination path of the folder in the managed folder
+        :param str folder: path  (absolute or relative) of the source folder to upload
+        """
+        real_root = os.path.realpath(folder)
```

Comment:
not sure if this is needed. It may actually make it more complex, I think os.walk will already natively return paths relative to the root

Reply:
Indeed, I will simplify it